topics/berry/index.md

```diff
@@ -0,0 +1,12 @@
+---
+aliases: Yarn 2
```

Comment:
Topic [`yarn 2`](https://github.com/topics/yarn-2) doesn't have many repositories but [`yarn2`](https://github.com/topics/yarn2) (w/o the space) has a few more so I think having just `yarn2` as an alias would be better.
```suggestion
aliases: yarn2
```

Reply:
seems fine to me! thanks.